### PR TITLE
Create country.js

### PR DESCRIPTION
**Pull request by JollyJolli**

### CODE REVIEW

**Comment by Leref** [comment]
Please copy the structure similar to https://github.com/Leref/aoi.js-library/blob/main/src/plugins/default/comment.js

the methods such `name, type, author, etc`.

As well putting the files you've created under your name, it will be same for author method.

**Review by JollyJolli — COMMENTED** [review]
👍

File: src/plugins/jollyjolli/country.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  );
+
+  return countryData;
+}
+
+// $country[country;format]
+
+module.exports = {
+  name: "$country",
+  type: "djs",
```
Comment (Leref) [inline src/plugins/jollyjolli/country.js:18]: Add missing metadata

**Review by Leref — CHANGES_REQUESTED** [review]

File: src/plugins/jollyjolli/country.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  const data = await response.json();
+
+  const countryData = data.countries.find(
+    (c) => c.name_en.toLowerCase() === country.toLowerCase() || c.name_es.toLowerCase() === country.toLowerCase()
+  );
+
+  return countryData;
+}
+
+// $country[country;format]
```
Comment (Leref) [inline src/plugins/jollyjolli/country.js:14]: Also not needed :)

**Review by Leref — APPROVED** [review]

**Pull request merged**